### PR TITLE
3.7.1 Add new migrations, rename tableName

### PR DESCRIPTION
**Pull request by IselDrew**

### CODE REVIEW

File: src/migrations/20200419200606-change-password-column.ts [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import { QueryInterface, DataTypes } from 'sequelize';
+import { Teachers } from '../modules/teachers/teachers.model';
+
+export async function up(query: QueryInterface) {
+  return query.changeColumn(Teachers.tableName, 'password', {
+    type: DataTypes.STRING(),
```
Comment (Alan18081) [inline src/migrations/20200419200606-change-password-column.ts:6]: Please add a constraint for the length of password

Comment (IselDrew) [reply src/migrations/20200419200606-change-password-column.ts]: fixed

File: src/migrations/20200419200606-change-password-column.ts [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import { QueryInterface, DataTypes } from 'sequelize';
+import { Teachers } from '../modules/teachers/teachers.model';
+
+export async function up(query: QueryInterface) {
+  return query.changeColumn(Teachers.tableName, 'password', {
+    type: DataTypes.STRING(50),
```
Comment (antonpanchenk0) [inline src/migrations/20200419200606-change-password-column.ts:6]: Mb string(255)?

Comment (IselDrew) [reply src/migrations/20200419200606-change-password-column.ts]: fixed

**Pull request merged**